src/Data/Container/Indexed/Examples/MultiSortedAlgebra.agda

```diff
@@ -0,0 +1,433 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- Multi-sorted algebras as indexed containers
+------------------------------------------------------------------------
+
+{-# OPTIONS --safe --without-K --guardedness #-}  -- forced by Agda-2.6.2-rc1
+
+module Data.Container.Indexed.Examples.MultiSortedAlgebra where
```

Comment:
I think we are now putting all `Examples` under the `README` directory. So I think this could be `README.Data.Container.Indexed.MultiSortedAlgebra`?

You should also import it in the `README.Data.Container.Indexed` module so that it gets type-checked.